src/harrastuspassi/harrastuspassi/api.py

```diff
@@ -75,10 +78,62 @@ def filter(self, qs, value):
         return super().filter(qs, list(values_with_children))
 
 
+class NearestOrderingFilter(filters.OrderingFilter):
+    def get_coordinates(self):
+        errors = defaultdict(list)
+        try:
+            near_latitude = self.parent.data.get('near_latitude')
```

Comment:
Does `get` raise KeyError here? What kind of data structure is `self.parent`? With a normal dictionary `get` just returns `None` without raising an exception is the key is not found.